ui/src/app/components/webcomponents/create-test-group-from-step-form.component.ts

```diff
@@ -84,6 +85,10 @@ export class CreateTestGroupFromStepFormComponent extends BaseComponent implemen
       setTimeout(() => this.searchInput.nativeElement.focus(), 200);
     })
   }
+  navigateToCase(testCaseId: number){
+    window.open('td/cases/'+testCaseId.toString(), '_blank');
+
```

Comment:
@Raksha-testsigma Remove the new line before closing braces and add a new line after before this function

Reply:
@tarun-testsigma made the changes bro. Please verify.